graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/stats/ClusterStatsApi.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.storage.elasticsearch7.stats;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.inject.Inject;
+import org.graylog.shaded.elasticsearch7.org.elasticsearch.client.Request;
+import org.graylog.storage.elasticsearch7.PlainJsonApi;
+import org.graylog2.rest.resources.system.indexer.responses.IndexSetStats;
+
+public class ClusterStatsApi {
+    private final ObjectMapper objectMapper;
+    private final PlainJsonApi jsonApi;
+
+    @Inject
+    public ClusterStatsApi(ObjectMapper objectMapper,
+                           PlainJsonApi jsonApi) {
+        this.objectMapper = objectMapper;
+        this.jsonApi = jsonApi;
+    }
+
+    public IndexSetStats clusterStats() {
+        final Request request = new Request("GET", "/_cluster/stats");
+        request.addParameter("filter_path", "indices.count, indices.docs.count, indices.store.size_in_bytes");
```

Comment:
Having whitespaces between the filter path arguments requires unnecessary URL escaping. While it seems to work, we shouldn't rely on it. (see the OpenSearch `ClusterStatsApi` as well)

```suggestion
        request.addParameter("filter_path", "indices.count,indices.docs.count,indices.store.size_in_bytes");
```

Reply:
Wasn't aware of that - good to know. Fixed it.